bgc_data/phytoinfo.dataresource.yaml

```diff
@@ -6,6 +6,8 @@ description: "A table of Phytoinfo"
 format: csv
 mediatype: text/csv
 encoding: utf-8
+layout:
+  skipFields: [ FID ]
```

Comment:
This is a column automatically added by Geoserver on output. We need to "skip" this field here because we don't actually care about it, and it is not included in the schema definition.